src/dataService/savedItemsService.ts

```diff
@@ -36,7 +37,7 @@ export class SavedItemDataService {
   private readonly userId: string;
   private readonly apiId: string;
 
-  constructor(context: IContext) {
+  constructor(context: Pick<IContext, 'dbClient' | 'userId' | 'apiId'>) {
```

Comment:
Simplifying the type here so that I don't have to create a whole context -- only what we actually need